test/e2e/network/netpol/model.go

```diff
@@ -71,10 +84,24 @@ func NewModel(namespaces []string, podNames []string, ports []int32, protocols [
 			var containers []*Container
 			for _, port := range ports {
 				for _, protocol := range protocols {
-					containers = append(containers, &Container{
```

Comment:
I don't understand this change -- isn't this just a more verbose way of doing the same thing?

Reply:
Yes, once we have a slice of protocols passed, we don't need this check